nominations/models.py

```diff
@@ -90,9 +91,23 @@ class NominationsPlatformAlert(models.Model):
         help_text='Show alert on nominations platform pages.'
     )
 
+    alert_level = models.CharField(
```

Comment:
Suggest to make this an integer field, to decouple the id we store in db from the actual css class that goes into the template.

Reply:
I don't know if that makes the most sense with use of enums, because we need to get the value. The value in this case is the css class. If it's an integer we'll have to do custom logic on the template side to translate the integers to a css value. Thoughts?

Reply:
We already have examples with more than 2 values per tuple in the Enum. You can do something like enum_member = (integer id, human display name, css class). See `class CallQuestion(Enum)`. And then the field looks like ```
    question = models.IntegerField(choices=[
        (x.value[0], x.value[1]) for x in CallQuestion
    ])
```